Sources/Fuzzilli/Modules/NetworkSync.swift

```diff
@@ -48,7 +48,7 @@ fileprivate protocol MessageHandler {
 /// A connection to a network peer that speaks the above protocol.
 fileprivate class Connection {
     /// The file descriptor on POSIX or SOCKET handle on Windows of the socket.
-    let socket: libsocket.socket_t
+    let socket: libsocket.socket_fd
```

Comment:
Not necessarily opposed to this rename, but why is it necessary?

Reply:
Either old or new version of macOS with the build of the swift just have encountered the error with bellow 
```shell
fuzzilli/Sources/libsocket/include/libsocket.h:39:13: note: candidate found by name lookup is 'socket_t'
typedef int socket_t;
            ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include/sys/kernel_types.h:96:41: note: candidate found by name lookup is 'socket_t'
typedef struct __socket*                socket_t;
```
Maybe there is no problem on Linux. So I suggest avoid the `reference to ' socket_t' is ambiguous` error